plugins/CoreConsole/Commands/DevelopmentSyncUITestScreenshots.php

```diff
@@ -11,15 +11,21 @@
 
 use Piwik\Development;
 use Piwik\Http;
+use Piwik\Log;
 use Piwik\Plugin\ConsoleCommand;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
+ * Tool for core developers to help making UI screenshot builds green.
+ *
  */
 class DevelopmentSyncUITestScreenshots extends ConsoleCommand
 {
+    protected $urlBase;
```

Comment:
I haven't looked in detail here but I agree with the idea: commands, controllers, services… should be stateless (e.g. not store state in properties).